src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": "0.1",
     "requires": {
-        "cockpit": "138"
+        "cockpit": "137"
```

Comment:
Not quite sure about this, since we changed quite a bit around 138

But it should be ok for the little while until fedora 26 is available as a good base.